build/media-suite_helper.sh

```diff
@@ -1273,6 +1273,8 @@ do_mesoninstall() {
 
 do_rust() {
     log "rust.update" "$RUSTUP_HOME/bin/cargo.exe" update
+    export SCCACHE_DIR=~/.sccache
```

Comment:
I'd rather `$HOME/.sccache`. I don't know if msys2 always expands the `~` correctly.

Reply:
```suggestion
    export SCCACHE_DIR=$HOME/.sccache
```